pkg/registry/apis/query/query.go

```diff
@@ -371,6 +375,25 @@ func (b *QueryAPIBuilder) handleExpressions(ctx context.Context, req parsedReque
 	return qdr, nil
 }
 
+func (b *QueryAPIBuilder) convertQueryWithoutExpression(ctx context.Context, req datasourceRequest,
+	qdr *backend.QueryDataResponse) (*backend.QueryDataResponse, error) {
+	allowLongFrames := false
+	refID := req.Request.Queries[0].RefID
+	_, results, err := b.converter.Convert(ctx, req.PluginId, qdr.Responses[refID].Frames, allowLongFrames)
```

Comment:
i'm not sure if this is the case here, but i've seen instances where the response is empty or missing some query responses. should this be checked

Reply:
Also added a check there.